walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -47,7 +47,7 @@
     Collections.sort(outOfRange, TimeRange.ORDER_BY_START);
 
     // iterate through and add timeranges to collection of available times
-    Collection<TimeRange> availableTimes = new ArrayList<>();
+    ArrayList<TimeRange> availableTimes = new ArrayList<>();
```

Comment:
nit: Is this change necessary?

Reply:
Yes, when I use Collection<TimeRange> availableTimes it gives me an error for .get() when I try to do TimeRange currentAvailableTime = availableTimes.get(i); at line 80.